base/abstractarraymath.jl

```diff
@@ -92,6 +92,10 @@ imag{T<:Real}(x::AbstractArray{T}) = zero(x)
 +{T<:Number}(x::AbstractArray{T}) = x
 *{T<:Number}(x::AbstractArray{T,2}) = x
 
+function lerpi(j::Integer, d::Integer, A::AbstractArray, B::AbstractArray)
```

Comment:
Is that in a way that lerpi over scalars and over arrays is in the same method and indistinguishable? Unless I'm really missing something I don't see why this method needs to be spelled the same way instead of inlining it out of existence.